src/styles/main.scss

```diff
@@ -0,0 +1,57 @@
+$width-box: 100px;
+$gap: 1px;
+$days-of-week: (mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6);
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 706px;
```

Comment:
The width of the `.calendar` block should not be hardcoded. It should be calculated based on the width of the day box and the gap size. This way, if you change the size of the box or the gap, the calendar will adjust automatically.